test/companies/schema/industry_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Companies.Schema.IndustryTest do
+  @moduledoc """
+    This module holds the unit tests to the industry changeset
+  """
+  use Companies.DataCase
+
+  alias Companies.{
+    Schema.Industry
+  }
+
+  @valid_industry %{
+    name: "Mining industry"
+  }
+
+
+  describe "industry changeset " do
```

Comment:
also here too many empty lines between `@valid_industry` and `describe`, there should be only one.

btw if you use VS Code you can install the [ElixirLS](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls). I find it very usefull as you can configure it to automatically format properly the code on save!

Reply:
@tajchumber I use Intellij mainly, but I will definitely checkout that VS code extension, thank you.